Workspace/src/ik/include/ik/Roboarm.h

```diff
@@ -1,26 +1,28 @@
 #ifndef ROBOARM_HEADER
 #define ROBOARM_HEADER
 
-
-class Roboarm{
-public:
-    float shoulder_pos;
-    float elbow_pos;
-    float wrist_pos;
-    const float shoulder_len;
-    const float elbow_len;
-    const float wrist_len;
-
-    float shoulder_vel;
-    float elbow_vel;
-    float wrist_vel;
-
-    void calc_velocities(float ee_x, float ee_y, float ee_phi);//updates joint velocities
-
-    Roboarm(float l_s, float l_e, float l_w,float shoulder=0, float elbow=0, float wrist=0);
-
+class Roboarm {
+  public:
+    void calc_velocities(float endEffectorVX, float endEffectorVY, float endEffectorPhi);
+
+    Roboarm(
+      float l1, float l2, float l3,
+      float a1 = 0, float a2 = 0, float a3 = 0
+      );
+    
     ~Roboarm();
-
-
+  private:
+    const float lengthLink1;
+    const float lengthLink2;
+    const float lengthLink3;
+    
+    float angleLink1;
+    float angleLink2;
+    float angleLink3;
+
+    float velocityLink1;
```

Comment:
I think that the velocities should be public. We will need to access them and publish them to an IK output topic